frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -93,34 +93,37 @@ export const updateDashboardAndCards = createThunkAction(
           .map(async dc => CardApi.update(dc.card)),
       );
 
-      // update the dashboard cards and tabs
       const dashcardsToUpdate = dashboard.dashcards.filter(dc => !dc.isRemoved);
-      const updatedCardsAndTabs = await DashboardApi.updateCardsAndTabs({
-        dashId: dashboard.id,
-        cards: dashcardsToUpdate.map(dc => ({
-          id: dc.id,
-          card_id: dc.card_id,
-          dashboard_tab_id: dc.dashboard_tab_id,
-          action_id: dc.action_id,
-          row: dc.row,
-          col: dc.col,
-          size_x: dc.size_x,
-          size_y: dc.size_y,
-          series: dc.series,
-          visualization_settings: dc.visualization_settings,
-          parameter_mappings: dc.parameter_mappings,
-        })),
-        ordered_tabs: (dashboard.ordered_tabs ?? [])
-          .filter(tab => !tab.isRemoved)
-          .map(({ id, name }) => ({
-            id,
-            name,
+      const updateCardsAndTabs = () =>
+        DashboardApi.updateCardsAndTabs({
+          dashId: dashboard.id,
+          cards: dashcardsToUpdate.map(dc => ({
+            id: dc.id,
+            card_id: dc.card_id,
+            dashboard_tab_id: dc.dashboard_tab_id,
+            action_id: dc.action_id,
+            row: dc.row,
+            col: dc.col,
+            size_x: dc.size_x,
+            size_y: dc.size_y,
+            series: dc.series,
+            visualization_settings: dc.visualization_settings,
+            parameter_mappings: dc.parameter_mappings,
           })),
-      });
+          ordered_tabs: (dashboard.ordered_tabs ?? [])
+            .filter(tab => !tab.isRemoved)
+            .map(({ id, name }) => ({
+              id,
+              name,
+            })),
+        });
+
+      // Make two parallel requests: one to update the dashboard and another for the dashcards and tabs
+      const [updatedCardsAndTabs, _updateDashboardAction] = await Promise.all([
```

Comment:
I don't see much point in getting the second variable if we're not going to use it. In most places, we do leave some to make sure people are aware of the returned value e.g. for old JS files that has specific API such as columns, visualization_settings, etc.

example 
https://github.com/metabase/metabase/blob/97f41c76831ef1210a67e762db8a202731cc0f96/enterprise/frontend/src/metabase-enterprise/auth/index.js#L49-L54

This might not be a good example, because the omitted variable is not the last used variable.